go.mod

```diff
@@ -27,6 +27,7 @@ require (
 	k8s.io/client-go v0.18.12
 	k8s.io/cloud-provider v0.0.0
 	k8s.io/component-base v0.0.0
+	k8s.io/klog v1.0.0
```

Comment:
we just updated to klog v2 (see line below), can you update this PR to use klog v2 pleaes?

Reply:
OK, I'm not sure how did I add this